src/components/FullscreenGrid/styles.module.scss

```diff
@@ -26,17 +26,35 @@ $expand-button-width: 40px;
   }
 }
 
+.Header,
+.topContent,
+.content,
+.mainSidePanel,
+.altSidePanel {
+  // Simulate a grid-gap.
+  margin-bottom: $default-padding;
+}
```

Comment:
Yeah, I couldn't figure out how to keep `grid-gap`. It's a bit of a bummer because `grid-gap` is hard to polyfill in a robust way. For our case, we got lucky because we mostly have only full width columns -- only the middle three need horizontal gaps. It's very fragile now. If we change the column layout we'll have to remember to make gaps.